### PR TITLE
Feat/#6

### LINKED ISSUE #6: Visual Footer Component Confirmation
### Explanation
It is located at the end of the page and seeks to provide the user with valuable information regarding  Azordev company.  For example, it contains the mission, the legal situation, the history, and etc.
The user can also find the login and register buttons, this with the purpose that if the person has not registered or has not logged in yet.

### Screenshots
image

### Checklist

- [ ]  Azordev company logo is highlighted on the left side of the footer.

- [ ] There is a variety  of information about the company, such as legality, which is one of the issues of mistrust in any other web page.

- [ ] There are two sections. The first one  deals with the information on the page and the second one is about who we are (Azordev).

- [ ] The login button is in a slightly dark blue #19456B color, with code letters #FFFFFF and the edges of the button are rounded at their tips.

- [ ] The sign up button is in a tertiary color code #11698E , with code letters #FFFFFF and the button edges are rounded at the ends.

### Step by step reproduction steps for the issue

- Open the app
- In any page scroll down to the bottom
- Verify the footer is different that in design 

### References

https://www.figma.com/file/zzEwuPfGTiZh4EA11MyZ5c/Sencico?node-id=65%3Aa

### PR DESCRIPTION
**Pull request by Allislove**
Added the new footer feature! #6

### CODE REVIEW

**Comment by Israel-Laguan** [comment]
@Allislove in mobile the buttons have different size

**Comment by e-azocar** [comment]
Please review this in desktop view
image

**Review by e-azocar — COMMENTED** [review]
@Allislove please review this in desktop view.

image

**Comment by Israel-Laguan** [comment]
I just checked, it has some kind of overflow on mobile devices
image

What about making it one column if the width is less than tablet view @angelik0828?

**Comment by Allislove** [comment]
Can you review please!

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -7,19 +7,22 @@ import { isAuthenticated } from "../utils/isAuthenticated";
...
+          >
+            Iniciar Sesión
+          </Link>
+          <Link
+            to="/register"
+            className="flex space-x-4  sm:inline-flex shadow-button items-end justify-center px-20 py-3.5 mb-5  text-xl font-semibold leading-6  xl:mb-8 md:mb-7 text-white whitespace-no-wrap bg-tertiary hover-scale rounded-md md:inline-flex xl:inline-flex mt-5 md:mt-4 mt-5 xl:mt-5"
+          >
+            Regístrate
+          </Link>
+
```
Comment (Israel-Laguan) [inline src/components/Footer.jsx]: Nit: this line can be erased

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -7,19 +7,22 @@ import { isAuthenticated } from "../utils/isAuthenticated";
...
+        <div className="flex flex-col sm:flex-inle sm:space-x-8 2xl:flex-col contents">
+          <Link
+            to="/login"
+            className="flex space-x-4  sm:inline-flex items-end justify-center px-20 py-3.5 mb-5  md:inline-flex xl:inline-flex 2xl:mb-7  md:mb-7  text-xl font-semibold leading-6 text-white whitespace-nowrap bg-primary rounded-md shadow-button hover-scale tracking-add mt-5 xl:mt-5"
+          >
+            Iniciar Sesión
+          </Link>
+          <Link
+            to="/register"
+            className="flex space-x-4  sm:inline-flex shadow-button items-end justify-center px-20 py-3.5 mb-5  text-xl font-semibold leading-6  xl:mb-8 md:mb-7 text-white whitespace-no-wrap bg-tertiary hover-scale rounded-md md:inline-flex xl:inline-flex mt-5 md:mt-4 mt-5 xl:mt-5"
```
Comment (Israel-Laguan) [inline src/components/Footer.jsx]: I notice some extra spaces between some styles

File: src/components/Footer.jsx (hunk 1 above)
Comment (Allislove) [reply src/components/Footer.jsx]: Done: what is Nit in this case ? XD:

File: src/components/Footer.jsx (hunk 2 above)
Comment (Allislove) [reply src/components/Footer.jsx]: Fixed!

File: src/components/Footer.jsx (hunk 1 above)
Comment (Israel-Laguan) [reply src/components/Footer.jsx]: Nit is short for "nitpicking", which meant someone is looking at really small details. In code reviews, the word refers to "I notice this small issue, its not that important but please check"

**Review by Israel-Laguan — APPROVED** [review]
@angelik0828 @e-azocar please check vissually

**Review by heliomar-pena — COMMENTED** [review]
I think the button on small devices should be centered, like the original design

imagen

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { isAuthenticated } from "../utils/isAuthenticated";
 const notLoggedButtons = () => {
   if (!isAuthenticated()) {
     return (
-      <div className="sm:col-span-2 sm:space-x-8 2xl:flex-col items-center justify-center xl:justify-between xl:px-10 2xl:px-0 2xl:justify-center sm:contents xl:contents 2xl:contents">
+      <div className="mx-20 sm:mx-12 md:mx-12 xl:mx-12 container mx-auto sm:col-span-2 sm:space-x-8 2xl:flex-col items-center justify-center xl:justify-between xl:px-10 2xl:px-0 2xl:justify-center sm:contents xl:contents 2xl:contents">
```
Comment (heliomar-pena) [inline src/components/Footer.jsx]:
```suggestion
      <div className="sm:mx-12 md:mx-12 xl:mx-12 container mx-auto sm:col-span-2 sm:space-x-8 2xl:flex-col items-center justify-center xl:justify-between xl:px-10 2xl:px-0 2xl:justify-center sm:contents xl:contents 2xl:contents">
```

mx-auto is overwriting mx-20 (or vice versa)

Comment (Allislove) [reply src/components/Footer.jsx]: Yeah. mx-auto was overwriting!

**Comment by Israel-Laguan** [comment]
@e-azocar @Katsu08 @angelik0828 please check

**Comment by angelik0828** [comment]
The best way to get to this view is to display a column, centered with the text aligned to the left.
something like that
image

**Comment by Allislove** [comment]
@angelik0828  Do you review the last change ?

**Review by e-azocar — APPROVED** [review]

**Review by heliomar-pena — APPROVED** [review]
Good work!

**Comment by Israel-Laguan** [comment]
The style proposed by @angelik0828 will be treated in a new issue, so this PR can be merged

**Comment by heliomar-pena** [comment]
@Israel-Laguan I have checked and is not a difficult change, I will add it to avoid to open an issue for so little change

**Pull request merged**